teamB/eunsong/db/migrations/20221005064612_create_posts_table.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE posts (
+		id INT NOT NULL AUTO_INCREMENT,
+		title VARCHAR(100) NOT NULL,
+		content VARCHAR(3000) NULL,
+		user_id INT NOT NULL,
+		created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+		updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
+		PRIMARY KEY(id),
+		CONSTRAINT posts_user_id_fkey FOREIGN KEY (user_id) REFERENCES users(id) 
+);
```

Comment:
더불어, `Indentation`을 맞춰서 작성해주세요 !
```suggestion
CREATE TABLE posts (
        id INT NOT NULL AUTO_INCREMENT,
        title VARCHAR(100) NOT NULL,
        content VARCHAR(3000) NULL,
        user_id INT NOT NULL,
        created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
        updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
        PRIMARY KEY(id),
        CONSTRAINT posts_user_id_fkey FOREIGN KEY (user_id) REFERENCES users(id) 
);
```